QUEST_JOBSTEP.tsv

```diff
@@ -2239,71 +2239,71 @@ QUEST_JOBSTEP_20150323_002238	I heard a solider at Fedimian saw the luggage of a
 QUEST_JOBSTEP_20150323_002239	This is the symbol that I engraved on my father's sleeve...{nl}This is so precious to me...Thank you.
 QUEST_JOBSTEP_20150323_002240	Just put this pot on Ramstis Valley.{nl}The problem is Chapparitions there.. You would be able to defeat it easily.
 QUEST_JOBSTEP_20150323_002241	Please bring me Hogma Spellers and Canines of Guards at Plateau of Rex.{nl}It won't be hard or you.{nl}
-QUEST_JOBSTEP_20150323_002242	I like your determination.{nl}Please defeat Sparnas Man at Vieta Valley to prove your skills.
-QUEST_JOBSTEP_20150323_002243	Cathapract is a warrior who stands in front of anyone and fight.{nl}If there are barriers in the way, just penetrate into them.{nl}I came with a good idea.
+QUEST_JOBSTEP_20150323_002242	I like your determination.{nl}Please defeat Sparnasman at Vieta Gorge to prove your skills.
+QUEST_JOBSTEP_20150323_002243	The Cathapract is a warrior who fights on the frontline.{nl}If there are barriers in the way, just penetrate into them.{nl}I came with a good idea.
 QUEST_JOBSTEP_20150323_002244	I heard monsters called Bitereginas are attacking tourists in Manacle Grave.{nl}I will acknowledge your skill if you could defeat those monsters.
 QUEST_JOBSTEP_20150323_002245	Contratulations for passing the test.{nl}My lessons are strict so I hope you follow well.
-QUEST_JOBSTEP_20150323_002246	Sparnas Man at Vieta Valley is said to be threatening people.{nl}He will not be an easy opponent, but he will be enough to prove your skills.
-QUEST_JOBSTEP_20150323_002247	The most important attribute Peltasta considers valuable is courage.{nl}Since Peltasta shouldn't avoid enemies' attacks and block them with their shields.
-QUEST_JOBSTEP_20150323_002248	Before you learn lessons from me, you should show me how brave you are.{nl}
-QUEST_JOBSTEP_20150323_002249	Defeat Chapparitions in Ramstis Valley.{nl}That bleakness will be appropriate to test your guts.
+QUEST_JOBSTEP_20150323_002246	Sparnasman at Vieta Gorge is said to be threatening people.{nl}He will not be an easy opponent, just right to prove your skills.
+QUEST_JOBSTEP_20150323_002247	The most important attribute the Peltastas consider valuable is courage.{nl}Since the Peltastas shouldn't avoid enemies' attacks but rather block them with their shields.
```

Comment:
This actually changes what is being conveyed somewhat.

Consider if it was "The most important attribute a Peltasta considers", and "Since a Peltasta shouldn't avoid enemies attacks" instead.  This is someone teaching another what the idea of a Peltasta is, something to aspire to, rather than referring to the current group of Peltastas.